src/ast.rs

```diff
@@ -75,6 +75,7 @@ pub enum StmtKind {
     Print(Expr),
     Melo(Iden),
     Rlyeh,
+    Rickroll
```

Comment:
Missing trailing comma - rustfmt

Reply:
Fun Fact - I hate trailing commas but I guess I'll comply with `rustfmt` here.

Reply:
Rustfmt exists to keep code readable and in a single style

Use it 😗👍

Reply:
I `!`(never) use trailing commas in my code :joy: